### ISSUE TITLE
warning: 'strncpy' specified bound depends on the length of the source argument [-Wstringop-overflow=]

### ISSUE THREAD

**Issue by dennisklein** [FairRootGroup/FairRoot#1052]
**OS**: Debian 10
**GCC**: 8.3.0

```
MbsAPI/fLmd.c: In function 'fLmdInitMbs':
MbsAPI/fLmd.c:427:9: warning: 'strncpy' specified bound depends on the length of the source argument [-Wstringop-overflow=]
         strncpy(pLmdControl->cFile, Nodename, len);
         ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
MbsAPI/fLmd.c:425:18: note: length computed here
     size_t len = strlen(Nodename);
                  ^~~~~~~~~~~~~~~~
```
```
MbsAPI/fLmd.c: In function 'fLmdPutOpen':
MbsAPI/fLmd.c:142:9: warning: 'strncpy' specified bound depends on the length of the source argument [-Wstringop-overflow=]
         strncpy(pLmdControl->cFile, Filename, len);
         ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
MbsAPI/fLmd.c:140:18: note: length computed here
     size_t len = strlen(Filename);
                  ^~~~~~~~~~~~~~~~
```
```
MbsAPI/fLmd.c: In function 'fLmdConnectMbs':
MbsAPI/fLmd.c:391:9: warning: 'strncpy' specified bound depends on the length of the source argument [-Wstringop-overflow=]
         strncpy(pLmdControl->cFile, Nodename, len);
         ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
MbsAPI/fLmd.c:389:18: note: length computed here
     size_t len = strlen(Nodename);
                  ^~~~~~~~~~~~~~~~
```
```
MbsAPI/fLmd.c: In function 'fLmdGetOpen':
MbsAPI/fLmd.c:590:9: warning: 'strncpy' specified bound depends on the length of the source argument [-Wstringop-overflow=]
         strncpy(pLmdControl->cFile, Filename, len);
         ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
MbsAPI/fLmd.c:588:18: note: length computed here
     size_t len = strlen(Filename);
                  ^~~~~~~~~~~~~~~~
```
```
MbsAPI/f_evt.c: In function 'f_evt_put_open':
MbsAPI/f_evt.c:1355:9: warning: 'strncpy' specified bound depends on the length of the source argument [-Wstringop-overflow=]
         strncpy(c_file, pc_file, len);
         ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~
MbsAPI/f_evt.c:1353:18: note: length computed here
     size_t len = strlen(pc_file);
                  ^~~~~~~~~~~~~~~
```
```
MbsAPI/f_evt.c:1399:21: warning: 'strncpy' specified bound depends on the length of the source argument [-Wstringop-overflow=]
                     strncpy(ps_file_head->filhe_user, username, len); /* user name */
                     ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
MbsAPI/f_evt.c:1396:30: note: length computed here
                 size_t len = strlen(username);
                              ^~~~~~~~~~~~~~~~
```

**Comment by ChristianTackeGSI** [comment]
Should be fixed in #1054

**Comment by ChristianTackeGSI** [comment]
Can we close this one?
* Should hopefully be fixed in  #1054
* We don't want to support MbsApi anyway.
